Packs/ExtraHop/ReleaseNotes/2_2_0.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ExtraHop Reveal(x)
+
+- %%UPDATE_RN%%
```

Comment:
Please describe here the general changes you made.

Reply:
This has been committed